aws/table_aws_backup_report_plan.go

```diff
@@ -0,0 +1,196 @@
+package aws
+
+import (
+	"context"
+	// "fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/backup"
+
+	// "github.com/aws/aws-sdk-go-v2/service/backup/types"
+
+	backupv1 "github.com/aws/aws-sdk-go/service/backup"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBackupReportPlan(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_backup_report_plan",
+		Description: "AWS Backup Report Plan",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValueException"}),
+			},
+			Hydrate: getAwsBackupReportPlan,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBackupReportPlans,
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(backupv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "arn",
+				Description: "An Amazon Resource Name (ARN) that uniquely identifies a resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ReportPlanArn"),
+			},
+			{
+				Name:        "name",
+				Description: "The unique name of the report plan.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ReportPlanName"),
```

Comment:
Is there any reason we changed the column name `report_plan_name` to `name`?
Usually, we do it for only `arn` column, rather than that we follow the snake case format for top-level properties of API response.